### PR TITLE
fix: remove authenticate when create data source

### PR DESCRIPTION
**Pull request by czy88840616**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: packages/sequelize/src/dataSourceManager.ts [hunk 1]
```diff
@@ -47,13 +47,16 @@ export class SequelizeDataSourceManager extends DataSourceManager<Sequelize> {
     const listEntities = listModule('sequelize:core');
     client.addModels(listEntities);
 
-    await client.authenticate();
+    const isConnected = this.checkConnected(client);
```
Comment (geekberryonekey) [inline packages/sequelize/src/dataSourceManager.ts:50]:
```
// const isConnected = this.checkConnected(client); 少了 await
const isConnected = await this.checkConnected(client);
```

Comment (czy88840616) [reply packages/sequelize/src/dataSourceManager.ts]: 恩，在写单测

File: packages/sequelize/test/index.test.ts [hunk 2]
```diff
@@ -11,48 +12,106 @@ function cleanFile(file) {
...
+                default: {
+                  database: 'test',
+                  username: 'test',
+                  password: 'test',
+                  dialect: 'mysql',
+                  host: '',
+                  port: 3306,
+                },
+              },
+              validateConnection: true,
```
Comment (geekberryonekey) [inline packages/sequelize/test/index.test.ts:107]: validateConnection 默认是 true 是不是更能保持版本间行为的一致性?

Comment (czy88840616) [reply packages/sequelize/test/index.test.ts:107]: 可以l

**Pull request merged**